ceph_medic/collector.py

```diff
@@ -122,6 +122,15 @@ def collect():
     failed_nodes = 0
     for node_type, nodes in cluster_nodes.items():
         for node in nodes:
+            # check if a node type exists for this node before doing any work:
+            try:
+                metadata[node_type]
+            except KeyError:
+                msg = "Skipping host group: {}".format(node_type)
+                terminal.warning(msg)
```

Comment:
I think this is fine just going to the file log. In the terminal it might be too much for someone that has many other hosts